### PR TITLE
noVNC console will show name of instance and scroll bars in case of resize

### PR DESCRIPTION
**Pull request by pawan-idrive**
As per the changes
- a new JS file is introduced, which will have custom code. Functions for this JS will be called after the connection happens
- now noVNC console will show the instance name and scroll bars will appear in the case of resize

Fixes idrive-online-backup/openstack-compute#596

### CODE REVIEW

File: vendor/idrive_vnc.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+  var body = document.getElementsByTagName('body')[0];
+  body.style.minWidth = width + 'px';
+  body.style.minHeight = height + 'px';
+  body.style.overflow = 'auto';
+  body.style.position = 'relative';
+
+  const urlParams = new URLSearchParams(window.location.search);
+  const myParam = urlParams.get('name');
+  document.title = myParam;
```
Comment (diganta-dutta) [inline vendor/idrive_vnc.js]: Update the `Connected to QEMU...` also

**Review by diganta-dutta — CHANGES_REQUESTED** [review]

File: vendor/idrive_vnc.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+/*
+// IDrive VNC Java Script file
+*/
+
+function handleConnect() {
+  var canvas = document.getElementsByTagName('canvas')[0];
+  canvas.focus();
+  canvas.style.paddingLeft = '10px';
```
Comment (diganta-dutta) [inline vendor/idrive_vnc.js]: @pawan-idrive why is the styling applied here? Can this not go into the css?

File: vnc_auto.html [hunk 3]
```diff
@@ -156,7 +157,7 @@
 <body>
     <div id="top_bar">
         <div id="status">Loading</div>
-        <div id="sendCtrlAltDelButton">Send CtrlAltDel</div>
+        <div id="sendCtrlAltDelButton">Send Ctrl+Alt+Del</div>
```
Comment (diganta-dutta) [inline vnc_auto.html]: I would prefer minimum necessary changes in this file.

**Review by diganta-dutta — CHANGES_REQUESTED** [review]

File: vendor/idrive_vnc.js (hunk 2 above)
Comment (pawan-idrive) [reply vendor/idrive_vnc.js]: Will move it

File: vnc_auto.html (hunk 3 above)
Comment (pawan-idrive) [reply vnc_auto.html]: Done

File: vendor/idrive_vnc.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  var winHeight = window.innerHeight;
+  var winWidth = window.innerWidth;
+
+  var topBar = document.getElementById('top_bar');
+  if ( winWidth > width ) {
+    topBar.style.minWidth = width + 'px';
+  } else {
+    topBar.style.minWidth = winWidth + 'px';
+  }
+  topBar.style.position = 'absolute';
```
Comment (diganta-dutta) [inline vendor/idrive_vnc.js:21]: More styling which can go to CSS

File: vendor/idrive_vnc.js [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    body.style.minWidth = width + 'px';
+  } else {
+    body.style.minWidth = winWidth + 'px';
+  }
+  if( winHeight > height ) {
+    body.style.minHeight = height + 'px';
+  } else {
+    body.style.minHeight = winHeight + 'px';
+  }
+  body.style.overflow = 'auto';
```
Comment (diganta-dutta) [inline vendor/idrive_vnc.js:35]: Can go to CSS

File: vendor/idrive_vnc.js [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+  var winWidth = window.innerWidth;
+
+  var topBar = document.getElementById('top_bar');
+  if ( winWidth > width ) {
+    topBar.style.minWidth = width + 'px';
+  } else {
+    topBar.style.minWidth = winWidth + 'px';
+  }
+  topBar.style.position = 'absolute';
+  topBar.style.bottom = 0;
```
Comment (diganta-dutta) [inline vendor/idrive_vnc.js]: Should go to CSS

File: vendor/idrive_vnc.js [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+  } else {
+    body.style.minWidth = winWidth + 'px';
+  }
+  if( winHeight > height ) {
+    body.style.minHeight = height + 'px';
+  } else {
+    body.style.minHeight = winHeight + 'px';
+  }
+  body.style.overflow = 'auto';
+  body.style.position = 'relative';
```
Comment (diganta-dutta) [inline vendor/idrive_vnc.js]: Should go to CSS. This is all styling.

**Review by diganta-dutta — APPROVED** [review]

**Pull request merged**